test/unit/ngsiv2/examples/contextRequests/updateContextAliasPlugin.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
In this case, due to you have a updateContextAliasPlugin2.json file also, this one could be named updateContextAliasPlugin1.json

Not too important... you can close this comment with NTC if you think it doesn't worth the time to change.

Reply:
(It seems this is the criteria you have followed with updateContextExpressionPlugin1.json and updateContextExpressionPlugin2.json)

Reply:
Fixed in 0b20a6e218960a8a2732a3dbf78d7c4ac76b52a5